native/tools/native-locales

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { program } = require('commander')
+
+const setupNativeLocales = (appName) => {
+  const { native: nativeLocales } = JSON.parse(fs.readFileSync('../../locales/locales.json'))
```

Comment:
This could be a problem because it is not clear form which current working directory this script should be run.
I think it makes sense to pass the locales.json file as paramter to this script. Then it is technically possible to run it from anywhere.